snippets/csharp/VS_Snippets_ADO.NET/DataWorks SqlConnectionStringBuilder.Remove/CS/source.cs

```diff
@@ -10,38 +10,29 @@ static void Main()
         try
         {
             string connectString =
-                "Data Source=(local);User ID=ab;Password= a1Pass@@11;" +
+                "Data Source=(local);User ID=ab;Password=myPassw0rd;" +
```

Comment:
I don't understand this change. Isn't this still bad?

Reply:
How about changing it to `SecureGetPassword()` ? That code is too easy to copy and paste. Customer can possibly use AKV to the the secret. On Prem customers can do something more secure.